src/components/MultipieceModal/MultipieceModal.test.js

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import {
+  fireEvent,
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+import {
+  componentPropsCheck,
+  getById,
+} from '../../../test/jest/helpers'
```

Comment:
Could you please add semicolon at the end?
Also could you please move this import after stripes components?

Reply:
done. thanks